server/include/triton/developer_tools/server_wrapper.h

```diff
@@ -410,7 +410,7 @@ InferRequest::AddInput(
 template <
     typename Iterator, typename std::enable_if<!std::is_same<
                            typename std::iterator_traits<Iterator>::value_type,
-                           std::string>::value>::type* = nullptr>
+                           std::string>::value>::type*>
```

Comment:
Could you please clarify why default value to the pointer changed?

Reply:
yeah, there was an error due to default arg being redefined in declaration
```
/opt/tritonserver/developer_tools/server/include/triton/developer_tools/server_wrapper.h:384:58: error: redefinition of default argument for 'typename std::enable_if<std::is_same<typename std::iterator_traits< <template-parameter-1-1> >::value_type, std::__cxx11::basic_string<char> >::value>::type* <anonymous>'
  384 |                            std::string>::value>::type* = nullptr>
      |                                                          ^~~~~~~
/opt/tritonserver/developer_tools/server/include/triton/developer_tools/server_wrapper.h:293:41: note: original definition appeared here
  293 |           std::string>::value>::type* = nullptr>
      |                                         ^~~~~~~
/opt/tritonserver/developer_tools/server/include/triton/developer_tools/server_wrapper.h:413:58: error: redefinition of default argument for 'typename std::enable_if<(! std::is_same<typename std::iterator_traits< <template-parameter-1-1> >::value_type, std::__cxx11::basic_string<char> >::value)>::type* <anonymous>'
  413 |                            std::string>::value>::type* = nullptr>
      |                                                          ^~~~~~~
/opt/tritonserver/developer_tools/server/include/triton/developer_tools/server_wrapper.h:316:41: note: original definition appeared here
  316 |           std::string>::value>::type* = nullptr>
      |    
```

Reply:
I see that default arg was given to iterator, it means that it expect that that value will be `nullptr` by default. 
As I see there is an `*` symbol which may correspond to the pointer. Pointer in C++ must be initialized.